test/integration/sellers-api-client.test.ts

```diff
@@ -60,17 +67,35 @@ describe(`${SellersApiClient.name}`, () => {
           secretAccessKey: Credentials?.SecretAccessKey || '',
           sessionToken: Credentials?.SessionToken || '',
         },
-      })
+      }
+    })
+
+    it('should return the list of marketplaces', async () => {
+      expect.assertions(1)
+
+      const client = new SellersApiClient(apiConfigurationParameters)
 
       const { data: marketplaceParticipations } = await client.getMarketplaceParticipations()
 
       expect(marketplaceParticipations.payload).toBeInstanceOf(Array)
     })
 
-    it('should assume role and return the list of marketplaces', async () => {
+    it('should return an array of reports', async () => {
       expect.assertions(1)
 
-      const tokens = await getTokens()
+      const client = new ReportsApiClient(apiConfigurationParameters)
+
+      const { data: reports } = await client.getReports({
+        reportTypes: ['GET_MERCHANT_LISTINGS_ALL_DATA'],
+      })
+
+      // TODO: replace with the commented out one when #75 is merged
```

Comment:
@nguyentoanit after merging #75, please fix this test and then merge the branch.